README.md

```diff
@@ -50,7 +65,7 @@ var options = {
   description: 'Credits towards consultation',
   image: 'https://i.imgur.com/3g7nmJC.png',
   currency: 'INR',
-  key: 'rzp_test_1DP5mmOlF5G5ag',
+  key: '', // your api key.
```

Comment:
@Nautiyalsachin Why we can't give test key for sample app. So they can check it without registering to Dashboard

Reply:
Because when they try to generate the key by registering to the Razorpay account, we will get some data like his phone number and email. And we can contact them again. Also, it will show us that a particular user came to dashboard and was not able to integrate with us. That will help with analytics too.

Reply:
Agree with that, but we making process more complicated. They just wanted to see app for that they have to registered to our Dashboard. User can not simply view application without registering. We can get data by how many hits for that particular test key to get data